features/config/osgi/cm/src/main/java/org/opennms/features/config/osgi/cm/MigratedServices.java

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2021 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 2021-2021 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.features.config.osgi.cm;
+import java.util.Set;
+
+// TODO: Patrick: find a better solution, e.g. read all available PIDs dynamically from CM.
+// TODO: Patrick: keep MigratedServices between 2 projects in sync
+public class MigratedServices {
+    final static Set<String> PIDS =  Set.of(
+            "org.opennms.features.datachoices"
```

Comment:
Is that possible to make org.opennms.features.config.osgi.cm depends on org.opennms.features.config.osgi.del ? So that you don't even need this class right ? If there is dependency issue just forget it.

Reply:
done.